controllers/home.js

```diff
@@ -1,10 +1,30 @@
 "use strict";
 
+const crypto = require("crypto");
 const { URL } = require("url");
+const uuidv4 = require("uuid/v4");
 
+const AppConstants = require("../app-constants");
 const HIBP = require("../hibp");
 const sha1 = require("../sha1-utils");
 
+
+function _generatePageToken(req) {
+  const pageToken = {ip: req.ip, date: new Date(), nonce: uuidv4()};
+  const cipher = crypto.createCipher("aes-256-cbc", AppConstants.COOKIE_SECRET);
```

Comment:
can we use `aes-256-gcm` instead?

Reply:
In that case we need to remember to update https://github.com/mozilla/blurts-server/pull/747/files#diff-3e18cca2cad1fa80c47052b459cd06a0R16 too

Reply:
Also, is the confidentiality of the token data important? Could we sign the data and submit nonce, date, and sig instead? I guess that might be larger.

Reply:
When I read [how to use GCM](https://lollyrock.com/posts/nodejs-encryption/), it looks like we would need a per-encryption IV, and we don't currently have a back-end data store for holding those IVs and associating them with each session ID. Is `-cbc` okay "for now" ?

Reply:
The IV and AAD should just be stored in cleartext alongside the encrypted data, but yes, cbc is fine for now.